pkg/util/util.go

```diff
@@ -11,6 +11,27 @@ type githubversion struct {
 	TagName string `json:"tag_name"`
 }
 
+const ConfigTemplate = `
+admin:
+  # For GRPC endpoints you might want to use dns:///flyte.myexample.com
+  endpoint: dns:///localhost:30081
+  insecure: true
+logger:
+  show-source: true
+  level: 3
+storage:
+  connection:
+    access-key: minio
+    auth-type: accesskey
+    disable-ssl: true
+    endpoint: http://localhost:30084
+    region: us-east-1
+    secret-key: miniostorage
+  type: minio
+  container: "my-s3-bucket"
+  enable-multicontainer: true
+`
```

Comment:
can we not use the config.yaml as the template instead of keeping both these.
I find people have to make changes in both the places which might not be likeable

Reply:
I think we can remove config.yaml from the repo and use init command